src/coreclr/tools/Common/JitInterface/CorInfoInstructionSet.cs

```diff
@@ -768,6 +768,54 @@ public InstructionSetInfo(string name, string managedName, InstructionSet instru
             }
         }
 
+        public static IEnumerable<string> AllCpuFamilies()
+        {
+            // Only report these in the help, others are added to CpuFamilyToInstructionSets just for muscle memory after clang/llvm
+            yield return "generic";
+            yield return "sandybridge";
+            yield return "haswell";
+            yield return "ampere";
```

Comment:
"ampere" is ambiguous.

Ampere is company name: https://amperecomputing.com/ . The company is producing number of different hardware models. We should use the hardware model name here.

Reply:
I agree, changed to ampere-altra which is probably also not a true name? I wasn't able to find the exact underlying CPU name? Was just wanted to offer something other than apple-m1 for arm64 especially for those who target [Azure](https://azure.microsoft.com/en-us/blog/now-in-preview-azure-virtual-machines-with-ampere-altra-armbased-processors/) with, at least, atomics being enabled.

Reply:
> especially for those who target [Azure](https://azure.microsoft.com/en-us/blog/now-in-preview-azure-virtual-machines-with-ampere-altra-armbased-processors/) with, at least, atomics being enabled.

There is no way to navigate and stay up to date on all these hardware codenames for 99+% of our customers. 

The easiest to understand alias would be `azure` that would map to Azure baseline at the time of the given .NET release, on both x64 and arm64. We can have similar alias for other clouds for parity.

Reply:
Makes sense, I'll leave it as is for now (without ampere) and clarify minimum baseline for azure hw